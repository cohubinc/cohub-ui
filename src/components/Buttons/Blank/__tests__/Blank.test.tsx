import React from "react";
import { mount } from "enzyme";

import { Buttons } from "dist";

describe("Blank", () => {
  it("renders without crashing", () => {
    const wrapper = mount(<Buttons.Blank>push me</Buttons.Blank>);
    expect(wrapper).toMatchSnapshot();
  });
});
