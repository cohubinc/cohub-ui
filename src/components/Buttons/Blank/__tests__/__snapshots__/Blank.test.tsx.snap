// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blank renders without crashing 1`] = `
<Blank>
  <button
    className="Blank-module_ButtonBlank__2u_7K "
  >
    <Typography
      style={Object {}}
    >
      <Component
        style={Object {}}
      >
        <span
          className=""
          data-qa="text"
          style={
            Object {
              "color": "hsl(0, 2%, 22%)",
              "fontFamily": "Inter",
              "fontSize": "14px",
              "fontWeight": 400,
              "textAlign": "initial",
              "textTransform": "initial",
            }
          }
        >
          push me
        </span>
      </Component>
    </Typography>
  </button>
</Blank>
`;
